tests/test.py

```diff
@@ -99,3 +99,16 @@ def test_mailto_bpdevs(page_url: tuple[Page, str]) -> None:
     page.goto(live_server_url)
     mailto = page.get_by_role("link", name="email")
     expect(mailto).to_have_attribute("href", "mailto:contact@blackpythondevs.com")
+
```

Comment:
Some tests that we have coded for that I would like to see.

#### Test 1:
- Add a small post as a demo (in test as a test case and destroy it after)
- Wait and let it rebuild and then check to see if the default image is grabbed.
 
 #### Test 2
 - If you select the next_button you expect the image to change to a new one. Same for prev_button
 
 #### Test 3
 - That slides match the links in the recent posts section
 
 #### Test 4
 - When you click the links in the carousel they go to the corresponding webpages

Reply:
Thanks Jay. I'll get to implementing these changes right away